### ISSUE TITLE
I have confirmed my suspicions.

### ISSUE THREAD

**Issue by lyallp** [openaudible/openaudible#1480]
I have confirmed my suspicions.
in the openaudible storage directory is a file called books.json
perform (in linux : jc is the json conversion tool) 
$ cat books.json | jc --indent 1 | grep -i unread

Then, I went into the Web export folder, manually edited the books,js to remove the "window.myBooks=" from the start of the file and saved it to temp.js
I then 
$ cat temp.js | jc --indent 1  |  grep -i unread 

The number of rows was 6 in my export to the web, and 22 in the books.js database used by OpenAudible.
I have just updated to the latest release of OpenAudible 4.4.8 on Windows 11 and re-exported my web page.

_Originally posted by @lyallp in https://github.com/openaudible/openaudible/issues/1474#issuecomment-2478456856_

**Comment by openaudible** [comment]
It looks like you're comparing the library's books.json file with the web directory books.js (which is identical to the web/books.json.)

The books.js only show books that were exported to the web directory. 

The web export skips some files, including book directories, ignored books, and podcasts. 

So if a book has a "directoryType" which might be podcast or audiobook, that "book" (or podcast) will be skipped entirely in the web export. 

The web export was an original feature that needs to be revamped for handling directory structures and podcasts. We are aware that it has a lot of limitations and recommend writing your own or using something like audiobookshelf.

**Comment by lyallp** [comment]
My point is, I am doing absolutely nothing fancy with Audible.
I buy audiobooks, I listen to them. Thatt's it.
No podcasts or anything else.
The reason why I noticed this was my annual subscription credits came up and I purchased 22 audiobooks in one hit.
They all successsfully appeared in the OpenAudible app as unread, but in the web page, only 6 appear as unread.

Are you suggesting that if an audiobook as an associated PDF, for example, that it does not export?

I am also happy to supply the json and js files as examples, if requested. (I am pretty sure they don't contain any account information :) )

If that is the case, or if there is some other already known factor, I am happy to accept that.

...Lyall

**Comment by openaudible** [comment]
PDF's are fine. Single file audiobooks are good. Split books are not. 

Why don't you look at the export of both json files and see what differences there are:
```
cat books.json | jc --indent 1 > b1.json
cat web/books.json | jc --indent 1 > b2.json
```
Is it really the unread status or is it the whole book is missing from the web page but not main library.
I just know that whatever read status we have (which might be blank, and not "unread") is copied over.

**Comment by openaudible** [comment]
Closing. Feel free to email us the files to info at openaudible dot org

**Comment by lyallp** [comment]
I discovered what the problem was.
At some point in the past, I exported the Web Page to the Open Audible installation webapp folder, and it created the books.js and books.json in THERE.

Later, I relocated the settings to create the Web Page on my NAS and when OpenAudible created the books.js and books.json on the NAS Server, it promptly overwrote them with the OLD copies I had created in my OpenAudible 'webapp' folder, giving the appearance of not including any new books!!

Removing the books.js and books.json from the 'OpenAudible' webapp folder fixed my issue.

Apologies for the misdirection.
